wire-ios-request-strategy/Tests/Sources/Mocks/MockMLSService.swift

```diff
@@ -105,4 +105,15 @@ class MockMLSService: MLSServiceInterface {
     func scheduleCommitPendingProposals(groupID: MLSGroupID, at commitDate: Date) {
         calls.scheduleCommitPendingProposals.append((groupID, commitDate))
     }
+
+    func createOrJoinSubgroup(
+        parentQualifiedID: QualifiedID,
+        parentID: MLSGroupID
+    ) async {
+
+    }
+
+    func generateConferenceInfo(for groupID: MLSGroupID) throws -> MLSConferenceInfo {
+        fatalError("not implemented")
+    }
 }
```

Comment:
Does this still make sense to have different implementations of same mocks across different frameworks?

Reply:
I want to generate this mock and reuse it but I thought to do it in another PR.